### DISCUSSION TITLE
en_core_web_lg does not break sentences

### DISCUSSION THREAD

**Discussion by wildfluss** [explosion/spaCy#12070] [Help: Coding & Implementations] [answered]
How come that `en_core_web_lg` does not break sentences at least as good as smaller one `en_core_web_sm`. It picks up "sentences" in the middle of ":" and other nonsense even 

```
import en_core_web_lg
import en_core_web_sm

lg = en_core_web_lg.load()
sm = en_core_web_sm.load()

txt = """IMAGE SOURCE: THE MOTLEY FOOL.

Arlington Asset Investment Corp  (AAIC)
Q1 2019 Earnings Call
May. 02, 2019, 9:00 a.m. ET

Contents:
Prepared Remarks
Questions and Answers
Call Participants
Prepared Remarks:

Operator

Good morning. I'd like to welcome everyone to the Arlington Asset First Quarter 2019 Earnings Call. Please be aware that each of your lines is in a listen-only mode. After the Company's remarks, we will open the floor for questions. (Operator Instructions)
"""

bad = lg(txt)

print(list(bad.sents))

good = sm(txt)

print(list(good.sents))

```

Will output this:

```
[IMAGE SOURCE:, THE MOTLEY FOOL.

, Arlington Asset Investment Corp  (AAIC)
, Q1 2019 Earnings Call
May., 02, 2019, 9:00 a.m., ET

Contents:
Prepared Remarks
Questions and Answers
Call Participants
Prepared Remarks:

Operator

Good morning., I'd like to welcome everyone to the Arlington Asset First Quarter 2019 Earnings Call., Please be aware that each of your lines is in a listen-only mode., After the Company's remarks, we will open the floor for questions., (Operator Instructions)
]
[IMAGE SOURCE: THE MOTLEY FOOL.

, Arlington Asset Investment Corp  (AAIC)
Q1 2019 Earnings Call
May. 02, 2019, 9:00 a.m., ET

Contents:
Prepared Remarks
Questions and Answers
...
Operator

Good morning., I'd like to welcome everyone to the Arlington Asset First Quarter 2019 Earnings Call., Please be aware that each of your lines is in a listen-only mode., After the Company's remarks, we will open the floor for questions., (Operator Instructions)
]
```

Versions:

```
$ pip show en_core_web_sm
Version: 3.4.1
$ pip show en_core_web_lg
Version: 3.4.1
```

**Comment by shadeMe** [comment, marked as answer]
The difference in accuracy can be attributed to the difference between your input text and the training data of the model in question: While the text above appears to be a transcript of a conversation, the large model was trained on [newspaper corpora](https://spacy.io/models/en#en_core_web_lg). As you can imagine, the structure and content of the former tends to differ significantly from that of the latter. 

For more context on how inputs influence the performance of a pre-trained model and tips to improving accuracy, please refer to [this post](https://github.com/explosion/spaCy/discussions/11952).
